src/actions/alerts/creators.js

```diff
@@ -0,0 +1,33 @@
+import * as t from './types';
+
+export function startLoading() {
+  return {
+    type: t.START_LOADING,
+  };
+}
+
+export function endLoading() {
+  return {
+    type: t.END_LOADING,
+  };
+}
+
+export function setError(message) {
+  return {
+    type: t.SET_ERROR,
+    payload: message,
+  };
+}
+
+export function setAlert(message) {
```

Comment:
we could use fewer actions and an alert-`level` instead (`level: 'error' || 'info'`)